src/main/java/com/revature/configs/SecurityConfiguration.java

```diff
@@ -65,6 +67,8 @@ protected void configure(HttpSecurity http) throws Exception {
         http.authorizeRequests()
                 // Our public endpoints
                 .antMatchers("/h2/**", "/login", "/register", "/signup", "/signup/shop", "/shop_products").permitAll()
+                .antMatchers(HttpMethod.GET, "/sellers/**").permitAll()
+                .antMatchers(HttpMethod.GET, "/invoices/**").permitAll()
```

Comment:
Getting invoices for sellers or for customers should require authentication so these shouldn't be being permitted.